webpack/JobWizard/steps/AdvancedFields/DescriptionField.js

```diff
@@ -25,6 +34,10 @@ export const DescriptionField = ({ inputs, value, setValue }) => {
   const [generatedDesc, setGeneratedDesc] = useState(generateDesc());
   const [isPreview, setIsPreview] = useState(true);
 
+  useEffect(() => {
+    setGeneratedDesc(generateDesc());
```

Comment:
does `generateDesc` need to `useCallback` with `inputValues` as a dependency?

Reply:
as useeffect's only dependency is inputValues i'm not sure if there is a difference between using useeffect with `generateDesc` as a dependency and what we have now

Reply:
I believe you should add `generateDesc` to the `useEffect` dependencies,
and then it should `useCallback` with `[inputValues, inputs, value]` as dependencies